src/loaders/MaterialLoader.js

```diff
@@ -100,6 +97,24 @@ Object.assign( MaterialLoader.prototype, {
 		if ( json.visible !== undefined ) material.visible = json.visible;
 		if ( json.userData !== undefined ) material.userData = json.userData;
 
+		// Shader Material
+
+		if ( json.uniforms !== undefined ) {
+
+			for ( var name in json.uniforms ) {
+
+				var uniform = json.uniforms[ name ];
+
+				material.uniforms[ name ] = {};
+				material.uniforms[ name ].value = ( uniform.type === 't' ) ? getTexture( uniform.value ) : uniform.value;
```

Comment:
I think we need Three.js object deserialization here. `value` can be `THREE.Color`, `THREE.Vector3`, `THREE.Matrix`, or so on.